internal/helm/values.go

```diff
@@ -102,7 +102,7 @@ var ciliumVals = map[string]map[string]any{
 		"endpointRoutes": map[string]any{
 			"enabled": true,
 		},
-		"extraArgs": []string{"--node-encryption-opt-out-labels=invalid.label"},
+		"extraArgs": []string{"--node-encryption-opt-out-labels=invalid.label", "--bpf-filter-priority=50"},
```

Comment:
We likely want this on all cloud, don't we? I know that this map/file likely needs some refactoring since there's a lot of duplicate configuration. For now, the easiest way would be just to copy the config.